dev/com.ibm.websphere.appserver.features/build.gradle

```diff
@@ -78,6 +78,14 @@ task copyFeaturePiiFiles {
           into rootProject.file('build.pii.package/nlssrc/' + featureName)
           include 'l10n/*.properties'
         }
+      } else {
+          if (project.file(featureProps['visibility'] + '/' + featureProps['symbolicName'] + '/resources').exists()) {
```

Comment:
With the switch to short name for the private / auto features, you need to update to short name here as well.

Reply:
visibility for an auto feature is private, so this will not work for auto features to find the files.